components/authz-service/storage/postgres/migration/sql/47_staged_rules_tables.up.sql

```diff
@@ -0,0 +1,94 @@
+BEGIN;
+
+CREATE TABLE iam_staged_projects (
+  db_id SERIAL,
+  id TEXT PRIMARY KEY,
+  name TEXT NOT NULL,
+  type iam_policy_type NOT NULL DEFAULT 'custom',
+  projects TEXT[],
+  deleted BOOLEAN,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+);
```

Comment:
We are staging projects? This is news to me but I guess it makes sense. Must have missed that.

Reply:
I just added it bc if a user deletes a project we'd want to trigger an update. . . right? or is that unrelated to this work?

Reply:
It's related to this work but we might want to tackle it after getting the rule staging right.

Reply:
well we'll put a pin it in but def worth thinking about as a separate task. I haven't seen it anywhere so we should at min make a card.